Caltech-IPAC/firefly
# DM-7803: add time series launch capability to suit app.

https://jira.lsstcorp.org/browse/DM-7803

On LSST PDAC, when searching for 'Deep Forced Source' data, there should be a button to view the time series data for the highlighted objectId.
If you upload a table with time series's metadata, there should be a button to view the table as time series.

To Test:
- Check out 'DM-7803_triview_timeseries' branch in both firefly and suit repo.
- Build suit and then goto http://localhost:8080/suit/
- Search for "Deep Forced Source", then click on 'View Time Series' button at upper right corner to view the data in a separate window.
- When uploading a table with time series metadata, i.e. 'timeCName', there should also be a 'View as Time Series' button at the same location
